PowerSimulationsPlots/src/call_plots.jl

```diff
@@ -78,6 +78,26 @@ PLOTLY_DEFAULT = vcat(
     CURTAILMENT_288,
 )
 
+function _filter_variables(results::PSI.Results; kwargs...)
+    filter_results = Dict()
+    reserves = get(kwargs, :reserves, false)
+    if reserves
+        for (key, var) in results.variables
+            if "$key"[1:2] == "P_" || "$key"[1:5] == "Spin_" || "$key"[1:4] == "Reg_" || "$key"[1:5] == "Flex_"
```

Comment:
This feels brittle. Are these strings coming from PSI const definitions for variables?  Seems like you could split by underscore and then compare the values.  And that could be made part of PSI.